main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -179,7 +179,15 @@ object Main {
     val flix = new Flix()
     flix.setOptions(options)
     for (file <- cmdOpts.files) {
-      flix.addPath(file.toPath)
+      val ext = file.getName.split('.').last
```

Comment:
What happens if there is no dot? Crash?

Reply:
Split then returns a singleton array with an empty string.

Reply:
The full string *

Reply:
OK

Reply:
So .last is always defined.